tests/attacks/evasion/test_robust_dpatch.py

```diff
@@ -0,0 +1,164 @@
+# MIT License
```

Comment:
Should we rename this file to `test_dpatch_robust.py` following the convention of prepending `test_` to the tested module's filename?

Reply:
I agree; I've renamed the file.